File: 1791-richest-customer-wealth/richest-customer-wealth.cs
public class Solution {
    public int MaximumWealth(int[][] accounts) {
        int richest = 0;
        foreach(int[] customer in accounts){
            int wealth = 0;
            foreach(int money in customer){
                wealth += money;
            }
            if (wealth > richest)
            richest = wealth;
        }
        return richest;
    }
}

File: 1791-richest-customer-wealth/richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int richest  = 0;
        for(int i = 0; i < accounts.length;i++){
            int wealth = Arrays.stream(accounts[i]).sum();
            if (wealth > richest)
            richest = wealth;
        }
        return richest;
    }
}